graphistry/pygraphistry.py

```diff
@@ -2448,6 +2450,72 @@ def _handle_api_response(response):
 
 
 
+    # Databricks Dashboard SSO helper functions
+
+    class DatabricksHelper():
+        
+        @staticmethod
+        def sso_repeat_get_token(repeat, wait):
```

Comment:
public methods should have .rst-friendly docstrs

Reply:
also, we're trying to make all new code mypy-typed

Reply:
ok, noted. I will add them in.